popup/popup.js

```diff
@@ -1,7 +1,7 @@
 
 window.onload = function() {
     document.getElementById('closeBtn').onclick = closeExtension;
-    document.getElementById('searchField').addEventListener('keyup', updateLocalStorage);
+    document.getElementById('searchField').addEventListener('input', updateLocalStorage);
```

Comment:
From testing I noticed that this does work well when closing application through close button, but when using keyboard shortcut the changes are not stored.